compiler-core/src/ast.rs

```diff
@@ -282,6 +289,47 @@ impl TypeAst {
             | TypeAst::Constructor { location, .. } => *location,
         }
     }
+
+    pub fn compare_without_location(&self, other: &TypeAst) -> bool {
+        match self {
+            TypeAst::Constructor {
+                module,
+                name,
+                arguments,
+                ..
```

Comment:
Can we replace the `..` with explicit `thingy: _`? I want to be alerted to future fields being added.